melos.yaml

```diff
@@ -68,12 +68,13 @@ scripts:
 
   analyze:
     run: melos exec -c 1 --fail-fast -- \
-      flutter analyze --no-fatal-infos
+      flutter analyze
```

Comment:
I like the idea and it would work fine for iOS/Flutter, but for Android it would need to build all the example apps before it's able to run the gradle script. I'll see if I can find a solution that doesn't involve building the apps.

Also, for iOS, the lint command is set to run as part of the Xcode build (without `--lint`) so that while you're developing, fixes are automatically applied. I'm not sure if Android has something similar (maybe with Gradle task deps), but that could be another solution as well.